src/java/org/apache/cassandra/db/DeletionTime.java

```diff
@@ -152,7 +156,7 @@ public boolean deletes(long timestamp)
 
     public int dataSize()
     {
-        return 12;
+        return 16;
```

Comment:
Since we're writing an int, I don't think this needs to change?

Reply:
`serializedSize()` in the serializers does indeed account for an int which is what gets written. This `dataSize()` is probably tied to the fact of holding ldt in a long which leads us to your other comment? I'll answer [there](https://github.com/apache/cassandra/pull/1891#pullrequestreview-1297370063)

Reply:
I believe {{dataSize()}} is an estimate of the size on disk, not size in memory.

Reply:
Confirmed with CI.